package.json

```diff
@@ -176,16 +176,17 @@
     "jest-junit": "^9.0.0",
     "jest-sentry-environment": "1.3.0",
     "prettier": "2.3.2",
+    "postcss-syntax": "0.36.2",
+    "postcss-jsx": "0.36.4",
     "react-refresh": "0.11.0",
     "react-select-event": "5.3.0",
     "react-test-renderer": "17.0.2",
     "size-limit": "^5.0.5",
     "speed-measure-webpack-plugin": "^1.5.0",
     "storybook-dark-mode": "^1.0.8",
-    "stylelint": "13.13.1",
-    "stylelint-config-prettier": "^8.0.2",
-    "stylelint-config-recommended": "^5.0.0",
-    "stylelint-config-styled-components": "^0.1.1",
```

Comment:
Does this mean stylelint no longer lints styles in styled components?

Reply:
from my understanding that package was to fix something temporarily https://github.com/styled-components/stylelint-config-styled-components#why

Reply:
i guess this is all it does? https://github.com/styled-components/stylelint-config-styled-components/blob/master/index.js